src/services/paris-requests.js

```diff
@@ -0,0 +1,7 @@
+import axios from "axios";
+
+const API_URL = "http://opendata.paris.fr/api/records/1.0/search/?dataset=stationnement-voie-publique-emplacements&q=&facet=regpri&facet=regpar&facet=typsta&facet=arrond&facet=zoneres&facet=tar&facet=locsta&facet=parite&facet=signhor&facet=signvert&facet=confsign&facet=typemob&facet=zoneasp&facet=stv&facet=prefet&facet=mtlast_edit_date_field&facet=datereleve&refine.regpar=V%C3%A9los";
+
+export const getParkingSpots = () => {
```

Comment:
Can you add the return Type to all of your methods ? 
Do you know why it could be important to always type your methods ?